plugins/filter_kubernetes/kubernetes.c

```diff
@@ -729,6 +729,13 @@ static struct flb_config_map config_map[] = {
      "Kubernetes authorization token file"
     },
 
+    /* Kubernetes Token command */
+    {
+     FLB_CONFIG_MAP_STR, "token_command", NULL,
```

Comment:
Can we rename it to `auth_token_command`?

Reply:
Rayhan's thinking makes sense... but Since we already have `Kube_Token_File` I think `Kube_Token_Command` makes more/the most sense.